__tests__/infinite_test.js

```diff
@@ -407,525 +364,489 @@ describe('The Behavior of the Variable Height React Infinite Component', functio
-        <Infinite elementHeight={elementHeight}
-                  containerHeight={800}
-                  timeScrollStateLastsForAfterUserScrolls={10000}
-                  className={"correct-class-name"}>
-          {renderHelpers.divGenerator(20, elementHeight)}
-        </Infinite>
-      );
-    var wrapper = rootNode.smoothScrollingWrapper;
-    expect(wrapper._style._values['pointer-events']).toBeUndefined();
+    const elementHeight = 200;
+    const rootNode = renderer.create(
+      <Infinite
+        elementHeight={elementHeight}
+        containerHeight={800}
+        timeScrollStateLastsForAfterUserScrolls={10000}
+        className={'correct-class-name'}
+      >
+        {renderHelpers.divGenerator(20, elementHeight)}
+      </Infinite>
+    );
+
+    expect(rootNode).toMatchSnapshot();
   });
 
   it('has pointer-events: none upon scroll', function() {
-    var infiniteSpy = jasmine.createSpy('infiniteSpy');
-    var elementHeight = 200;
-    var rootNode = TestUtils.renderIntoDocument(
-        <Infinite elementHeight={elementHeight}
-                  containerHeight={800}
-                  timeScrollStateLastsForAfterUserScrolls={10000}
-                  className={"correct-class-name"}>
-          {renderHelpers.divGenerator(20, elementHeight)}
-        </Infinite>
-      );
+    const elementHeight = 200;
+    const rootNode = mount(
+      <Infinite
+        elementHeight={elementHeight}
+        containerHeight={800}
+        timeScrollStateLastsForAfterUserScrolls={10000}
+        className={'correct-class-name'}
+      >
+        {renderHelpers.divGenerator(20, elementHeight)}
+      </Infinite>
+    );
 
-    var rootDomNode = ReactDOM.findDOMNode(rootNode);
+    const rootDomNode = rootNode.getDOMNode();
     rootDomNode.scrollTop = 100;
-    TestUtils.Simulate.scroll(rootDomNode, {
+    rootNode.simulate('scroll', {
       target: rootDomNode
     });
 
-    var wrapper = rootNode.smoothScrollingWrapper;
-    expect(wrapper._style._values['pointer-events']).toEqual('none');
+    expect(mountToJson(rootNode)).toMatchSnapshot();
   });
 });
 
 describe('Handling infinite scrolling', function() {
   it('triggers an infinite scroll the first time the component mounts if the elements do not fill the container', function() {
-    var infiniteSpy = jasmine.createSpy('infiniteSpy');
-    var elementHeight = 200;
-    var rootNode;
-
-    runs(function() {
-      rootNode = TestUtils.renderIntoDocument(
-        <Infinite elementHeight={elementHeight}
-                  containerHeight={800}
-                  infiniteLoadBeginEdgeOffset={1000}
-                  onInfiniteLoad={infiniteSpy}
-                  timeScrollStateLastsForAfterUserScrolls={10000}
-                  className={"correct-class-name"}>
-        </Infinite>
-      );
-    });
-
-    waitsFor(function() {
-      return infiniteSpy.callCount > 0;
-    });
+    const infiniteSpy = jasmine.createSpy('infiniteSpy');
```

Comment:
jasmine APIs are [no longer officially supported within jest](https://github.com/facebook/jest/pull/3147), so for future compatibility these should be replaced with `jest.fn()`